pkgs/development/python-modules/marisa-python/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchgit, marisa, swig }:
+
+buildPythonPackage rec {
+  pname = "marisa-python";
+  version = "1.3.40";
+  name = "${pname}-${version}";
```

Comment:
No `name`.

Reply:
Sorry, I don't get it. Why don't you like 6th line where `name` is defined?

Reply:
There's no need for `name`. It is generated from `pname` and `version` by `buildPythonPackage`.

Reply:
Got it. Thanks for advice.